sound/soc/sof/trace.c

```diff
@@ -182,6 +204,7 @@ int snd_sof_init_trace_ipc(struct snd_sof_dev *sdev)
 	}
 
 	sdev->dtrace_is_enabled = true;
+	sdev->dtrace_eof = true;
```

Comment:
is it necessary to initialize dtrace_eof here, as well as .....

Reply:
@plbossart So this is the stupid mistake I made. Sorry about that again. That should have been "sdev->dtrace_eof = false" -- when FW trace is active, we have no need to signal EOF of course.